pkg/provider/browser/browser.go

```diff
@@ -21,11 +21,21 @@ type Client struct {
 
 // New create new browser based client
 func New(idpAccount *cfg.IDPAccount) (*Client, error) {
-	return &Client{Headless: idpAccount.Headless}, nil
+	return &Client{
+		Headless: idpAccount.Headless,
+	}, nil
 }
 
 func (cl *Client) Authenticate(loginDetails *creds.LoginDetails) (string, error) {
 
+	// Optionally download browser drivers if specified
+	if loginDetails.DownloadBrowser {
```

Comment:
will this (re)download/install at every run?

Reply:
It shouldn't re-download since playwright-go does have that caching mechanism. Just by manually invoking the relevant `playwright install` command it doesn't re-download the binary, at least not without first nuking the relevant cache.

With this code, if you were to set it in the config file, it would functionally bring you back to the 2.36.4 behavior where it always checks if you have the drivers or not. If you do, it outputs the relevant message.

Alternatively, if you don't set it in the config, it will only download the driver if you set the relevant flag.